values.yaml

```diff
@@ -9,9 +9,14 @@ images:
 # source-to-image builder images, each language represents a distinct ecosystem the builder supports,
 # sometimes it includes a specific version
 s2iBuilders:
-  go: docker.io/centos/go-toolset-7-centos7:latest
-  python: quay.io/sclorg/python-39-c9s:latest
-
+  
+  go: registry.redhat.io/rhel8/go-toolset:1.19.9-2             #docker.io/centos/go-toolset-7-centos7:latest
+  python: registry.access.redhat.com/ubi8/python-39:latest     #quay.io/sclorg/python-39-c9s:latest
+  ruby: registry.access.redhat.com/ubi9/ruby-31:1-50           #registry.access.redhat.com/ubi8/ruby-31:1-51
+  perl: registry.access.redhat.com/ubi9/perl-532:1-91
+  php: registry.access.redhat.com/ubi9/php-81:1-29
+  java: registry.redhat.io/redhat-openjdk-18/openjdk18-openshift:1.15-6.1687264188 #registry.access.redhat.com/ubi8/openjdk-11:latest   
```

Comment:
nit: please remove the empty line (12).

not-so-nit: let's not leave comments with previous values, please 🙏 Let move this information, as we need, to issues on the project.